tests/features/session_tracking.feature

```diff
@@ -27,3 +27,37 @@ Scenario: Set Auto Capture Sessions sends
     When I run "SessionSetAutoCaptureScenario"
     And I wait to receive a request
     Then the request is valid for the session reporting API version "1.0" for the "Android Bugsnag Notifier" notifier
+
+Scenario: User is persisted between sessions
```

Comment:
Do these scenarios also need to be added to the local feature which uses mazerunner v1?

Reply:
This should be added in now